### ISSUE TITLE
Cannot read property 'quickSearches' of undefined

### ISSUE THREAD

**Issue by Christilut** [mingo-app/mingo#234]
![image](https://user-images.githubusercontent.com/3471363/106274522-af12d300-6234-11eb-8c9e-949751e08a11.png)

Not sure if related, but loading the first 50 documents is taking quite a long time:

![image](https://user-images.githubusercontent.com/3471363/106274593-cbaf0b00-6234-11eb-98ee-598392518c07.png)

Says 37ms, but it took about 20 seconds until the results started showing. Showed a loader during that time.

**Comment by Christilut** [comment]
Looks like the quickSearches error happens when I press CTRL+G (or press Paste&Go). I have a document ID on my clipboard.

**Comment by tothradoslav** [comment]
First about the quickSearches error: Could you try refreshing the database or entire connection? (top left sync icon in the sidebar) This is due to migration of data to this new version of Mingo.

Regarding the 20 seconds issue: I have noticed this with large collections (1+ million documents), but in my case it was an issue with the server itself (haven't accessed the collection for a while and took him some time to load the documents). The next load was fast. Could you confirm this or does it happen all the time?

**Comment by Christilut** [comment]
It's still slow after a refresh and every time I try it

**Comment by Christilut** [comment]
The quickSearches error doesn't happen anymore after the refresh

**Comment by tothradoslav** [comment]
ok, is that a large collection? Or large documents?

**Comment by tothradoslav** [comment]
.countDocuments() was causing the slow response. This command is slow on large collections. We used this due to a previous issue with .count() which wasn't exact. So we either use a precise number but will be slow, or we use an approximate number which is very fast. 

.countDocuments() is also every CPU demanding...

We use the total number of documents of a query to show and use it to manipulate multiple documents. The problem is, that if the number is not "precise", we may have pagination issues. 

I used the .count() for now, it's very fast.
